manifests/profile/docker.pp

```diff
@@ -66,7 +66,8 @@
     include docker
   } else {
     class { 'docker':
-      version => $version,
+      version          => $version,
+      extra_parameters => ['--insecure-registry=hatcher-kubernetes.umdl.umich.edu:32030'],
```

Comment:
This is a hack to let us have a private registry immediately. Longer-term, if we want a private registry, it should at the very least run over HTTPS so that we don't need this line. The one that's up is minimal and only exists so that we can have an insightiq pod today.